app/infrastructure/persistence/rls_check.py

```
@@ -0,0 +1,116 @@
+
+    try:
+        import asyncpg
+    except ImportError:
+        return RLSCheckResult(
+            ok=False,
+            message="asyncpg is required for RLS check. Install with: uv sync",
+        )
+
+    try:
+        conn = await asyncpg.connect(conn_url)
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)